timezone.gemspec

```diff
@@ -29,6 +29,6 @@ Gem::Specification.new do |s|
 
   s.add_development_dependency('minitest', '~> 5.8')
   s.add_development_dependency('rake')
-  s.add_development_dependency('rubocop', '= 0.51')
+  s.add_development_dependency('rubocop', '~> 0.52', '>= 0.52.1')
```

Comment:
I would prefer to not allow minor (possibly patch) version bumps for RuboCop. As new cops are added or modified they can often require code changes to comply. Upgrades to this particular dev dependency should be manual. Happy with an upgrade to `= 0.52.1` if you want to do that.

Reply:
>  As new cops are added or modified they can often require code changes to comply.

With a non-strict version locking it'll be noticeable. Code can be changed with new PRs, instead of manually monitoring new RuboCop versions and change it in those PRs. Old version of code is working, just can be a bit better.

Reply:
I would still prefer to keep this fully specified - and manually bump the version every once and a while. From the perspective of an outside contributor, I would not want to be required to correct un-related code simply because I happened to create a PR after a RuboCop version bump and before the `master` branch had been updated.

Reply:
> I would still prefer to keep this fully specified - and manually bump the version every once and a while.

It seems to me problematic to manually keep track of new versions.

> From the perspective of an outside contributor, I would not want to be required to correct un-related code simply because I happened to create a PR after a RuboCop version bump and before the master branch had been updated.

Why do you think that outside contributor will be required to correct unrelated code? You'll require it? I see you don't want to do this. And I don't see any RuboCop-related required checks for PRs.

Reply:
RuboCop checks run as part of CI: https://github.com/panthomakos/timezone/blob/master/Rakefile#L19. Even if there is a patch or minor upgrade in RuboCop that usually means that either a new rule has been added or a new formatting case has been caught. I would rather handle those cases in separate PRs.

Reply:
OK, I didn't know about Travis CI. As you wish. Also, you can think about `after_failure` in Travis CI.